apps/web-giddh/src/assets/css/global/vat-report.scss

```diff
@@ -1,7 +1,45 @@
 .vat-report {
     .vat-report-table {
+        &.mat-mdc-table {
+            &.zimbabwe-vat-report-table {
+                tr {
+                    height: 44px !important;
+
+                    th {
+                        padding: 5px 10px !important;
+                    }
+
+                    th,td {
+                        border:  1px solid var(--color-table-primary-content-bar-border) !important;
+        
+                        &[colspan="2"]{
+                            text-align: center;
+                        }
+                        &.d-none {
+                            display: none !important;
+                        }
+                        &.bg-grey {
+                            background-color: var(--color-wild-sand) !important;
+                        }
+                    }
+                }
+                tbody {
+                    tr {
+                        &:last-child {
+                            display: none !important;
+                        }
+                    }
+                }
+            }
+        }
+        
         tfoot {
             background-color: var(--color-wild-sand);
+            tr {
+                td {
```

Comment:
for font-weight 700 
use bootstrap4 class font-weight-bold
bootstrap5 fw-bold

Reply:
Done